distributed/comm/__init__.py

```diff
@@ -23,7 +25,7 @@ def _register_transports():
 
     tcp_backend = dask.config.get("distributed.comm.tcp.backend")
 
-    if tcp_backend == "asyncio":
+    if sys.platform == "win32" or tcp_backend == "asyncio":
```

Comment:
FWIW I'm fine with this. To the best of my knowledge, the asyncio backend is not used anywhere